### PR TITLE
ENG-10447: Changes method to clone demo notebooks

### PR DESCRIPTION
**Pull request by bredamc**
ENG-10447: Changes method to clone demo notebooks from CodeFlare SDK

### CODE REVIEW

File: modules/running-distributed-data-science-workloads-from-notebooks.adoc [hunk 1]
```diff
@@ -68,14 +68,38 @@ The examples in this procedure refer to the JupyterLab integrated development en
...
-To access the demo notebooks, clone the `codeflare-sdk` repository as follows:
-
-.. In the JupyterLab interface, click *Git > Clone a Repository*.
-.. In the "Clone a repo" dialog, enter `https://github.com/project-codeflare/codeflare-sdk.git` and then click *Clone*.
-The *codeflare-sdk* repository is listed in the left navigation pane.
-. Locate the downloaded demo notebooks as follows:
-.. In the JupyterLab interface, in the left navigation pane, double-click *codeflare-sdk*.
-.. Double-click *demo-notebooks*, and then double-click *guided-demos*.
+To access the demo notebooks, run the `codeflare_sdk.copy_demo_nbs()` function in a Jupyter notebook.
+The function identifies the demo notebooks that are compatible with the currently installed version of the CodeFlare SDK, and clones them into the target location.
```
Comment (KPostOffice) [inline modules/running-distributed-data-science-workloads-from-notebooks.adoc:72]:
```suggestion
The function copies the demo notebooks that are packaged with the currently installed version of the CodeFlare SDK, and clones them into the target location. 
```

**Review by KPostOffice — COMMENTED** [review]
Just one small change

File: modules/running-distributed-data-science-workloads-from-notebooks.adoc (hunk 1 above)
Comment (bredamc) [reply modules/running-distributed-data-science-workloads-from-notebooks.adoc]: Done, thanks!

File: modules/running-distributed-data-science-workloads-from-notebooks.adoc [hunk 2]
```diff
@@ -68,14 +68,38 @@ The examples in this procedure refer to the JupyterLab integrated development en
...
++
+The `codeflare_sdk.copy_demo_nbs()` function accepts two arguments:
+
+* `dir(str)` specifies the target location for the cloned demo notebooks. 
+If the leading character is a forward slash (/), the path is absolute. 
+Otherwise, the path is relative to the current directory.
+The default value is `"./demo-notebooks"`.
+
+* `overwrite(bool)` allows the function to overwrite the contents of an existing directory.
+When this argument is set to `True`, the function overwrites any customizations that you made in the original demo files.
```
Comment (chtyler) [inline modules/running-distributed-data-science-workloads-from-notebooks.adoc:82]: I'm just wondering if you we can cut out "that you made" here.

**Review by chtyler — APPROVED** [review]
Approved, but with a small suggestion.

File: modules/running-distributed-data-science-workloads-from-notebooks.adoc [hunk 3]
```diff
@@ -68,14 +68,38 @@ The examples in this procedure refer to the JupyterLab integrated development en
...
++
+----
+codeflare_sdk.copy_demo_nbs("my-dir", True)
+codeflare_sdk.copy_demo_nbs("my-dir")
+codeflare_sdk.copy_demo_nbs(dir="my-dir")
+codeflare_sdk.copy_demo_nbs(overwrite=True)
+codeflare_sdk.copy_demo_nbs(dir="my-dir", overwrite=True)
+codeflare_sdk.copy_demo_nbs(overwrite=True, dir="my-dir")
+codeflare_sdk.copy_demo_nbs("/absolute/path/to/target-dir")
+----
```
Comment (jiridanek) [inline modules/running-distributed-data-science-workloads-from-notebooks.adoc:98]: do we expect the user to be familiar with Python? lot's of the examples are just variations on Python syntax (such as that keyword arguments can be given in any order). I'd cut down on the number of examples here

```suggestion
----
codeflare_sdk.copy_demo_nbs("my-dir")
codeflare_sdk.copy_demo_nbs("/absolute/path/to/target-dir")
codeflare_sdk.copy_demo_nbs(dir="my-dir", overwrite=True)
----
```

File: modules/running-distributed-data-science-workloads-from-notebooks.adoc [hunk 4]
```diff
@@ -68,14 +68,38 @@ The examples in this procedure refer to the JupyterLab integrated development en
...
-The *codeflare-sdk* repository is listed in the left navigation pane.
-. Locate the downloaded demo notebooks as follows:
-.. In the JupyterLab interface, in the left navigation pane, double-click *codeflare-sdk*.
-.. Double-click *demo-notebooks*, and then double-click *guided-demos*.
+To access the demo notebooks, run the `codeflare_sdk.copy_demo_nbs()` function in a Jupyter notebook.
+The function copies the demo notebooks that are packaged with the currently installed version of the CodeFlare SDK, and clones them into the target location. 
++
+The `codeflare_sdk.copy_demo_nbs()` function accepts two arguments:
+
+* `dir(str)` specifies the target location for the cloned demo notebooks.
```
Comment (jiridanek) [inline modules/running-distributed-data-science-workloads-from-notebooks.adoc:76]:
```suggestion
* `dir: str` specifies the target location for the cloned demo notebooks. 
```

that's the usual Python convention for specifying a type

**Review by jiridanek — APPROVED** [review]

**Pull request merged**